code/modules/uplink/uplink_items/uplink_implants.dm

```diff
@@ -12,91 +12,97 @@
-
-/datum/uplink_item/implants/microbomb
-	name = "Microbomb Implant"
-	desc = "An implant injected into the body, and later activated either manually or automatically upon death. \
-			The more implants inside of you, the higher the explosive power. \
-			This will permanently destroy your body, however."
-	item = /obj/item/storage/box/syndie_kit/imp_microbomb
-	cost = 2
-	include_modes = list(/datum/game_mode/nuclear)
-
-/datum/uplink_item/implants/macrobomb
-	name = "Macrobomb Implant"
-	desc = "An implant injected into the body, and later activated either manually or automatically upon death. \
-			Upon death, releases a massive explosion that will wipe out everything nearby."
-	item = /obj/item/storage/box/syndie_kit/imp_macrobomb
-	cost = 20
-	include_modes = list(/datum/game_mode/nuclear)
-	restricted = TRUE
+	cost = 4
 
 /datum/uplink_item/implants/radio
 	name = "Internal Syndicate Radio Implant"
 	desc = "An implant injected into the body, allowing the use of an internal Syndicate radio. \
 			Used just like a regular headset, but can be disabled to use external headsets normally and to avoid detection."
 	item = /obj/item/storage/box/syndie_kit/imp_radio
-	cost = 4
+	cost = 3
 	restricted = TRUE
 
-/datum/uplink_item/implants/reviver
-	name = "Reviver Implant"
-	desc = "This implant will attempt to revive and heal you if you lose consciousness. Comes with an autosurgeon."
-	item = /obj/item/autosurgeon/reviver
-	cost = 8
-	include_modes = list(/datum/game_mode/nuclear)
+/datum/uplink_item/implants/uplink
+	name = "Uplink Implant"
+	desc = "An implant injected into the body, and later activated at the user's will. Has no telecrystals and must be charged by the use of physical telecrystals. \
+			Undetectable (except via surgery), and excellent for escaping confinement."
+	item = /obj/item/storage/box/syndie_kit/imp_uplink
+	cost = 3
+	// An empty uplink is kinda useless.
+	surplus = 0
+	restricted = TRUE
 
 /datum/uplink_item/implants/stealthimplant
 	name = "Stealth Implant"
 	desc = "This one-of-a-kind implant will make you almost invisible as long as you don't don't excessively move around. \
 			On activation, it will conceal you inside a chameleon cardboard box that is only revealed once someone bumps into it."
 	item = /obj/item/implanter/stealth
-	cost = 8
+	cost = 6
 
 /datum/uplink_item/implants/storage
 	name = "Storage Implant"
 	desc = "An implant injected into the body, and later activated at the user's will. It will open a small bluespace \
-			pocket capable of storing two regular-sized items."
+			pocket capable of storing two regular-sized items. You may only purchase one."
 	item = /obj/item/storage/box/syndie_kit/imp_storage
-	cost = 8
+	limited_stock = 1
+	cost = 6
+
+/datum/uplink_item/implants/emp
+	name = "EMP Implant"
+	desc = "An implant injected into the body, and later activated at the user's will. It will create a small EMP blast around the user."
+	item = /obj/item/storage/box/syndie_kit/imp_emp
+	cost = 1
```

Comment:
EMP doesn't need buffing

Reply:
This isn't a buff, it only comes with the Implant, nades are a seperate item.

Reply:
It is a buff, making EMPs easier to access is a straight buff, regardless of separation.

Reply:
What the fuck does this even mean. The 2 tc price tag was always worth it. The only reason they got merged was because it was messy. It would not matter if it was the bundled package or not, they are still absolutely worth the tc. In fact it's a little weird they would be separated again since if you only brought the kit doe the implant you miss out on the incredible value of those grenades.